pkg/securitypolicy/securitypolicyenforcer.go

```diff
@@ -87,7 +87,7 @@ func newSecurityPolicyFromBase64JSON(base64EncodedPolicy string) (*SecurityPolic
 	securityPolicy := new(SecurityPolicy)
 	err = json.Unmarshal(jsonPolicy, securityPolicy)
 	if err != nil {
-		return nil, errors.Wrap(err, "unable to unmarshal JSON policy")
+		return nil, errors.Wrapf(err, "unable to unmarshal JSON policy: %v", jsonPolicy)
```

Comment:
Is this supposed to be in here?

Reply:
I had added and for testing and figured it would be fine to keep for debugging purposes in the future, but in retrospect I should remove it.